assets/js/hale-scripts.js

```diff
@@ -14,3 +14,54 @@ if( document.getElementById("search-show-hide") != null) {
     }
   );
 }
+
+jQuery( document ).ready(function( $ ) {
+
+  //Keeps the Quick Exit Button at top of page once scrolled past the buttons original position in header
+  $(window).on('scroll resize', function() {
+    var stickyDiv = $('.govuk-exit-this-page');
+    var stickyButton = $('.govuk-exit-this-page a');
+
+    if (stickyDiv.length) {
+
+      //Keeps the surrounding Div the same width and height even when the button is fixed position. As this is used to work out the buttons left pos.
+      var divMinHeight = stickyButton.outerHeight();
```

Comment:
Is it worth adding in the unit as well?  var divMinHeight = stickyButton.outerHeight() + 'px';